content/docs/csidriver/features/powerflex.md

```diff
@@ -522,6 +522,66 @@ Then run:
 this test deploys the pod with two ephemeral volumes, and write some data to them before deleting the pod.   
 When creating ephemeral volumes, it is important to specify the following within the volumeAttributes section: volumeName, size, storagepool, and if you want to use a non-default array, systemID.  
 
+## Consuming Existing Volumes with Static Provisioning
+
+You can use existent volumes from PowerFlex array as Persistent Volumes in your Kubernetes, perform the following steps:
+1. Open your volume in the PowerFlex Management UI. Filter by just your volume and the volume ID can be located in the URL.
+2. Keep track of your system ID and volume ID as they will be needed in the following steps.
```

Comment:
Replace "in the following steps" with "during this procedure"

Reply:
Updated.